domain/autocert/state/state_test.go

```diff
@@ -277,5 +277,5 @@ Hn+GmxZA
 
 	// Retrieve the non-existent cert.
 	_, err = st.Get(context.Background(), "cert1")
-	c.Assert(err, gc.ErrorMatches, "autocert cert1: not found")
+	c.Assert(err, gc.ErrorMatches, "acme/autocert: certificate cache miss")
```

Comment:
Same comment as above around asserting type not string.

Reply:
done